stock_reserve/tests/test_stock_reserve.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields
+from odoo.tests import Form, common
+
+
+class TestStockReserve(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.env["res.config.settings"].create(
+            {
+                "group_stock_multi_locations": True,
+            }
+        ).execute()
```

Comment:
```suggestion
        ).set_values()
```
never use `execute` unless you really need to. It triggers unnecessary machinery.